homeassistant/components/blueprint/schemas.py

```diff
@@ -68,7 +93,7 @@ def validate_yaml_suffix(value: str) -> str:
         vol.Required(CONF_USE_BLUEPRINT): vol.Schema(
             {
                 vol.Required(CONF_PATH): vol.All(cv.path, validate_yaml_suffix),
-                vol.Required(CONF_INPUT): {str: cv.match_all},
+                vol.Required(CONF_INPUT, default=dict): {str: cv.match_all},
```

Comment:
We want to provide an empty dict as default, is that correct?
```suggestion
                vol.Required(CONF_INPUT, default={}): {str: cv.match_all},
```

Reply:
If `default` is a callable, voluptuous uses it to create a new instance. So we want it to be an empty dict that is unique for each invocation of the validator.

Reply:
Makes sense!

Here's the source that checks that:
https://github.com/alecthomas/voluptuous/blob/de695d855949b44ec1cce41f243ad9c6a39034fd/voluptuous/schema_builder.py#L134-L137